css-flexbox-practice.css

```diff
@@ -0,0 +1,102 @@
+.main {
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+}
+.site {
+    display: flex;
+    flex-direction: column;
+    width: 90%;
+    max-width: 800px;
+}
+
+.navigation {
+    display: flex;
+    justify-content: space-between;
+}
+
+.nav-list {
+    list-style-type: none;
+    display: flex;
+}
+
+.list-item {
+    margin-left: 30px;
+}
+
+.section-1{
+    display: flex;
+    justify-content: space-between;
+    border: 1px solid black;
+    margin-bottom: 50px;
+}
+
+.section-1-left {
+    display: flex;
+    width: 25%;
+}
+
+.section-1-right {
+    width: 75%;
+    align-items: center;
```

Comment:
You can only use align-items on a flex container.... (i.e. - `display:flex` is present)